_observing-your-data/sso.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: default
+title: Simple Schema for Observability
+nav_order: 120
+---
+
+# Simple Schema for Observability
+Introduced 2.6
+{: .label .label-purple }
+
+Starting with OpenSearch 2.6, a standardization for conforming to a common and unified observability schema has been introduced: Simple Schema for Observability (SSO). [Observability]({{site.url}}{{site.baseurl}}/observing-your-data/index/) is collection of plugins and applications that let you visualize data-driven events by using Piped Processing Language to explore, discover, and query data stored in OpenSearch.
+
+With the schema in place, observability tools can ingest and automatically extract and aggregate data and create custom common dashboards making it easier to understand the system at a higher level.
+
+The SSO schema is inspired by both [OTEL](https://opentelemetry.io/docs/) and Elastic Common Schema (ECS) and uses the large variety that ECS has to offer for logs with the OTEL metadata.
+
+Alerts will be supported in a future release.
+{: .note }
+
+### Use cases
+
+Use cases for SSO include:
+
+- Ingest observability data from different data types.
```

Comment:
Revise bullets to make them parallel. Suggested revisions:

- Ingesting...
- Moving from non-transferable, proprietary configurations to consolidated, sharable solutions that allow users...and have capabilities to display data analyses.
- Conforming Data Prepper and SSO for metrics, with gradual support of traces and logs.